DQMOffline/Configuration/python/DQMOffline_SecondStep_cff.py

```diff
@@ -45,8 +45,7 @@
 from DQMOffline.JetMET.SusyPostProcessor_cff import *
 from DQMOffline.JetMET.dataCertificationJetMET_cff import *
 from DQM.TrackingMonitorClient.TrackingClientConfig_Tier0_cff import *
-from DQM.TrackingMonitorClient.pixelTrackingEffFromHitPattern_cff import *
```

Comment:
this looks like unintended inversion of #22395

Reply:
Oh yes, I will fix this appropriately